docs/mint.json

```diff
@@ -56,6 +56,7 @@
     {
       "group": "Tutorials",
       "pages": [
+        "tutorials/common-errors",
```

Comment:
We might want a better section for this doc. For example, it can be in the same section with this doc: https://docs.mage.ai/development/troubleshooting

Reply:
Good call, missed that section